usr/src/lib/libsmbfs/smb/print.c

```diff
@@ -33,6 +32,10 @@
  * $Id: print.c,v 1.1.1.3 2001/07/06 22:38:43 conrad Exp $
  */
 
+/*
+ * Copyright 2018 Nexenta Systems, Inc.  All rights reserved.
```

Comment:
I'm stunned that this file still exists.

Reply:
it still works :)